pkg/apis/networking/types.go

```diff
@@ -398,19 +400,51 @@ type HTTPIngressRuleValue struct {
 	// options usable by a loadbalancer, like http keep-alive.
 }
 
-// HTTPIngressPath associates a path regex with a backend. Incoming urls matching
-// the path are forwarded to the backend.
+// PathType represents the type of path referred to by a HTTPIngressPath.
+type PathType string
+
+const (
+	// PathTypeExact matches the URL path exactly.
```

Comment:
does this include case-sensitivity (e.g. would a path of `/FOO` match a request to `/foo` or vice-versa)?

does this include %-encoding (e.g. would a path of `/foo bar` match a request to `/foo%20bar`?

Reply:
I have updated this to clarify that both exact and prefix matches are case sensitive. I think the %-encoding is a great question, but likely too significant in scope to handle here. A brief survey of implementations shows that behavior here varies already, and since we have not yet had a chance to develop consensus around a specific approach here, I'd rather leave it unspecified until we can. I will work on getting that consensus in place prior to a v1 release.

Reply:
sounds good.